test/api/organization/update.test.js

```diff
@@ -0,0 +1,120 @@
+const test = require('ava')
+const { before, beforeEach, afterEach, after } = require('../../_helpers/_setup')
+const { USER_WEB_SESSION_COOKIE } = require('../../../helpers/constants')
+
+test.before(async (t) => {
+  await before(t, async ({ db, auth }) => {
+    const email = 'honey@etsy.com'
+    const { id: userId1 } = await db.user.create({ email })
+    t.context.userId1 = userId1.toHexString()
+
+    const { id: orgId1 } = await db.organization.create({
+      name: 'flossbank',
+      host: 'GitHub',
+      userId: t.context.userId1,
+      avatarUrl: 'blah.com',
+      email
+    })
+    t.context.orgId1 = orgId1.toString()
+    await db.organization.updateCustomerId({ orgId: t.context.orgId1, customerId: 'honesty-cust-id' })
+    await db.organization.setDonation({ orgId: t.context.orgId1, amount: 1000, globalDonation: false })
+
+    const sessionWithDonation = await auth.user.createWebSession({ userId: t.context.userId1 })
+    t.context.sessionWithDonation = sessionWithDonation.sessionId
+  })
+})
+
+test.beforeEach(async (t) => {
+  await beforeEach(t)
+})
+
+test.afterEach(async (t) => {
+  await afterEach(t)
+})
+
+test.after.always(async (t) => {
+  await after(t)
+})
+
+test('PUT `/organization` 401 | unauthorized', async (t) => {
```

Comment:
add test for 400 -- missing org id or missing email. pretty sure that's why it built despite using old schema